### PR TITLE
basics/loops: add foreach example + postfix iteration

### PR DESCRIPTION
**Pull request by wilzbach**

### CODE REVIEW

File: basics/loops.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ D provides four loop constructs.
 The classical `for` loop known from C/C++ or Java
 with _initializer_, _loop condition_ and _loop statement_:
 
-    for (int i = 0; i < arr.length; ++i) {
+    for (int i = 0; i < arr.length; i++) {
```
Comment (wilzbach) [inline basics/loops.md]: in comparison to C++, there's no difference and this looks visually more appealing (to me)

File: basics/loops.md [hunk 2]
```diff
@@ -30,10 +30,14 @@ evaluated for the first time.
         foo();
     } while (condition);
```
Comment (wilzbach) [inline basics/loops.md]: It's a bit bizarre that we use two different style for the loops (the `for`-loop uses and example, whereas `while uses pseudo-style)

File: basics/loops.md [hunk 3]
```diff
@@ -30,10 +30,14 @@ evaluated for the first time.
...
 
-#### 4) `foreach`
+### 4) `foreach`
 
-The [`foreach` loop](basics/foreach) which will be introduced in the
-next section.
+The [`foreach` loop](basics/foreach) which will be introduced in more details
+in the next section.
+
+    for (el; arr) {
```
Comment (ZombineDev) [inline basics/loops.md]: foreach

**Comment by ZombineDev** [comment]
BTW, `while` loops should be before `for`, because they're simpler.

**Pull request merged**